microsoft-edge/devtools-guide-chromium/css/inspect.md

```diff
@@ -0,0 +1,82 @@
+---
+description: Learn how to use Microsoft Edge DevTools to analyze HTML pages using the Inspect tool 
+title: Inspect Tool in Microsoft Edge DevTools
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 12/16/2021
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools
+---
+
+# Analyze HTML pages using the Inspect tool 
+
+This article shows you how to use the **Inspect** tool to preview information about and select an element from the current document. You can try it out yourself by checking the [Inspect Demo Page](https://microsoftedge.github.io/DevToolsSamples/inspector/inspector-demo.html).
+
+The **Inspect** tool button (![Inspect](../media/inspect-icon.msft.png)) is in the upper-left corner of DevTools.  When you select the **Inspect** tool button, the button turns blue, indicating that the **Inspect** tool is active.
```

Comment:
It looks like you reused an existing icon from another page. The problem with this is that the icon in your next screenshot is different from this one, and that may be confusing.

I know it feels like having a shared library of icons would be better, but we actually prefer to have images be per page instead. Even if that means having several copies of the same image.
Could you please create a new screenshot of the inspect icon just for this article?

The other related thing here is: we normally use the default theme for all screenshots. Do you think it'd be ok to take those screenshots again but using the Light+ theme instead?